pytradfri/api/aiocoap_api.py

```diff
@@ -26,6 +32,7 @@ def __init__(self, host: str, psk_id="pytradfri", psk=None, internal_create=None
         self._observations_err_callbacks = []
```

Comment:
I've tried removing this list, since we don't use it, but then for some reasons the error callbacks are not called by aiocoap. By storing the callbacks aiocoap calls them on shutdown. :man_shrugging: